corehq/apps/app_manager/tests/test_suite_multi_select_case_list.py

```diff
@@ -40,8 +41,33 @@ def setUp(self):
         module.assign_references()
 
     def test_multi_select_case_list(self):
+        suite = self.factory.app.create_suite()
+        self.assertXmlPartialEqual(
+            """
+<partial>
+  <entry>
+    <form>some-xmlns</form>
+    <command id="m0-f0">
+      <text>
+        <locale id="forms.m0f0"/>
+      </text>
+    </command>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <session>
+      <datum id="case_id"
+             nodeset="instance('casedb')/casedb/case[@case_type='person'][@status='open']"
+             value="./@case_id"
+             detail-select="m0_case_short"
+             detail-confirm="m0_case_long"/>
+    </session>
+  </entry>
+</partial>
```

Comment:
is the lack of indentation necessary?

Reply:
Just avoiding line length issues, but it looks like that isn't necessary. Updated in 4ac76e21a9a